ElementX/Sources/UITests/UITestsAppCoordinator.swift

```diff
@@ -50,7 +50,7 @@ class UITestsAppCoordinator: AppCoordinatorProtocol {
             delegate.window??.layer.speed = 0
         }
         
-        guard let screenID = Tests.screenID else { fatalError("Unable to launch with unknown screen.") }
+        guard let screenID = ProcessInfo.screenID else { fatalError("Unable to launch with unknown screen.") }
```

Comment:
Should probably rename this one to be a bit more explicit that its for tests.
```suggestion
        guard let screenID = ProcessInfo.testScreenID else { fatalError("Unable to launch with unknown screen.") }
```